src/libostree/ostree-repo.c

```diff
@@ -542,6 +593,7 @@ ostree_repo_lock_push (OstreeRepo          *self,
 /**
  * ostree_repo_lock_pop:
  * @self: a #OstreeRepo
+ * @lock_type: the type of lock to release
```

Comment:
Might be worth pointing out that it’s a programmer error to release the wrong type of lock, compared to what your program acquired in the first place.

Reply:
:+1:

Reply:
I put this in the description text.